NEWS.md

```diff
@@ -44,130 +45,133 @@
-   + added argument `allow_empty_lines` (`FALSE` by default) to toggle this behavior
-   + improved so that trailing whitespace inside string literals does not trigger a lint (#1045, @AshesITR)
-   * added argument `allow_in_strings` (`TRUE` by default) to toggle this behavior
+   + Extended to also lint completely blank lines by default (#1044, @AshesITR)
+   + Added argument `allow_empty_lines` (`FALSE` by default) to toggle this behavior
+   + Improved so that trailing whitespace inside string literals does not trigger a lint (#1045, @AshesITR)
+   + Added argument `allow_in_strings` (`TRUE` by default) to toggle this behavior
 * `trailing_blank_lines_linter()`: extend to lint files without a terminal newline (#675, @AshesITR)
+   + Also, running `lint()` on a file without a terminal newline no longer throws a `warning()`
 * `object_name_linter()`
-   + improved generic detection -- in user-defined method `my_method.upstream.class`,
+   + Improved generic detection -- in user-defined method `my_method.upstream.class`,
      `upstream.class` no longer throws a lint because the generic (`my_method`)
      properly uses `snake_case` (#737, @AshesITR)
-   + exclude special R namespace hook functions such as `.onLoad` (#500, #614, @AshesITR and @michaelchirico)
-   + correctly detect imported functions when linting packages (#642, @AshesITR)
-   + correctly detect assignment generics like `names<-.class_name` (#843, @jonkeane)
-   + added new styles `"symbols"` and `"SNAKE_CASE"` (#494, #495, #615, #670, @michaelchirico and @AshesITR)
+   + Exclude special R namespace hook functions such as `.onLoad` (#500, #614, @AshesITR and @MichaelChirico)
+   + Correctly detect imported functions when linting packages (#642, @AshesITR)
+   + Correctly detect assignment generics like `names<-.class_name` (#843, @jonkeane)
+   + Added new styles `"symbols"` and `"SNAKE_CASE"` (#494, #495, #615, #670, @MichaelChirico and @AshesITR)
+     - `"symbols"` is a new default style which won't lint all-symbol object names. In particular, that means
+       operator names like `%+%` are skipped.
+   + No longer lints names used in `$` extractions (#582, @AshesITR)
 * `object_usage_linter()`
-   + detect global variables if there are top-level dollar-assignments (#666, #709, @AshesITR)
-   + report usage warnings spanning multiple lines (#507, @AshesITR)
-   + detect usages inside `glue::glue()` constructs (#942, @AshesITR)
-   + detect within functions assigned with `=` instead of `<-` (#1081, @michaelchirico)
-   + detect functions exported by packages that are explicitly attached using `library()` or
+   + Detect global variables if there are top-level dollar-assignments (#666, @AshesITR)
+   + Report usage warnings spanning multiple lines (#507, @AshesITR)
+   + Detect usages inside `glue::glue()` constructs (#942, @AshesITR)
+   + Detect within functions assigned with `=` instead of `<-` (#1081, @MichaelChirico)
```

Comment:
Runs on functions assigned with ... ?

Reply:
changed to 'Detect usages inside' to be consistent with the previous `glue()` point

Reply:
The `glue()` point regards extracting variable usages. Doesn't this point mean we run `checkUsage()` on more functions?

Reply:
That's what #1081 was about, right?

Reply:
Yes so it should be "Checks usage within functions assigned ..." or "Runs on functions assigned ...".
#942 is about avoiding false positives.

Reply:
Agree about distinguishing the bullets, don't like those phrasings though. Went with 'Extended to include functions'